lib/ibandit/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Ibandit
-  VERSION = "0.11.28".freeze
+  VERSION = "1.0.0"
```

Comment:
should we be cutting a new version in this PR?

Reply:
In the PR or at all? Ivan's comment in https://github.com/gocardless/ibandit/pull/120 makes it seem like its required.